### PR TITLE
better handling on removing global.conf

### PR DESCRIPTION
**Pull request by Groovylein**
Found an bad line during installation in #803 
Replaced it with checking first before deleting

### CODE REVIEW

File: scripts/installscripts/buster-install-default.sh [hunk 1]
```diff
@@ -861,7 +861,12 @@ sudo amixer cset numid=1 100%
 # delete the global.conf file, in case somebody manually copied stuff back and forth
 # this will be created the first time the Phoniebox is put to use
 # by web app or RFID
-rm /home/pi/RPi-Jukebox-RFID/settings/global.conf
+GLOBAL_CONF=/home/pi/RFID/settings/global.conf
+if [ -f $GLOBAL_CONF ]
+then
+   echo "global.conf need to be deleted."
```
Comment (s-martin) [inline scripts/installscripts/buster-install-default.sh]: "need" --> "needs"

File: scripts/installscripts/buster-install-mopidy3.sh [hunk 2]
```diff
@@ -820,7 +820,13 @@ sudo amixer cset numid=1 100%
 # delete the global.conf file, in case somebody manually copied stuff back and forth
 # this will be created the first time the Phoniebox is put to use
 # by web app or RFID
-rm /home/pi/RPi-Jukebox-RFID/settings/global.conf
+
+GLOBAL_CONF=/home/pi/RFID/settings/global.conf
+if [ -f $GLOBAL_CONF ]
+then
+   echo "global.conf need to be deleted."
```
Comment (s-martin) [inline scripts/installscripts/buster-install-mopidy3.sh]: "need" --> "needs"

File: scripts/installscripts/buster-install-default.sh (hunk 1 above)
Comment (Groovylein) [reply scripts/installscripts/buster-install-default.sh]: created a new commit 5fd9a2b

File: scripts/installscripts/buster-install-mopidy3.sh (hunk 2 above)
Comment (Groovylein) [reply scripts/installscripts/buster-install-mopidy3.sh]: created a new commit 5fd9a2b

File: scripts/installscripts/buster-install-default.sh [hunk 3]
```diff
@@ -861,7 +861,12 @@ sudo amixer cset numid=1 100%
 # delete the global.conf file, in case somebody manually copied stuff back and forth
 # this will be created the first time the Phoniebox is put to use
 # by web app or RFID
-rm /home/pi/RPi-Jukebox-RFID/settings/global.conf
+GLOBAL_CONF=/home/pi/RFID/settings/global.conf
```
Comment (s-martin) [inline scripts/installscripts/buster-install-default.sh]: Sorry, didn't catch that when I first looked at it, but the path is not correct, it should be `/home/pi/RPi-Jukebox-RFID/settings/global.conf`

File: scripts/installscripts/buster-install-mopidy3.sh [hunk 4]
```diff
@@ -820,7 +820,13 @@ sudo amixer cset numid=1 100%
 # delete the global.conf file, in case somebody manually copied stuff back and forth
 # this will be created the first time the Phoniebox is put to use
 # by web app or RFID
-rm /home/pi/RPi-Jukebox-RFID/settings/global.conf
+
+GLOBAL_CONF=/home/pi/RFID/settings/global.conf
```
Comment (s-martin) [inline scripts/installscripts/buster-install-mopidy3.sh]: see comment above

File: scripts/installscripts/buster-install-default.sh (hunk 3 above)
Comment (Groovylein) [reply scripts/installscripts/buster-install-default.sh]: Oh yeah, you are right. I will fix that. Stupid copy-paste failure.

Comment (Groovylein) [reply scripts/installscripts/buster-install-default.sh]: Updated with 4116795

File: scripts/installscripts/buster-install-mopidy3.sh (hunk 4 above)
Comment (Groovylein) [reply scripts/installscripts/buster-install-mopidy3.sh]: Updated with 4116795

**Pull request merged**